code/workspaces/web-app/src/components/notebooks/CreateNotebookForm.spec.js

```diff
@@ -52,6 +17,7 @@ describe('CreateNotebookForm', () => {
       { text: 'First Data Store', value: 'alpha' },
       { text: 'Second Data Store', value: 'beta' },
     ],
+    projectKey: 'project',
```

Comment:
Would it be worth using 'testproj' or something, rather than 'project', otherwise a hardcoded 'project' is harder to see.

Reply:
updated as suggested